raiden/tests/utils/eth_node.py

```diff
@@ -510,14 +472,25 @@ def run_private_blockchain(
             seal_account=seal_account,
         )
 
+        for config in nodes_configuration:
+            if config.get("mine"):
+                datadir = eth_node_to_datadir(config["nodekeyhex"], base_datadir)
+                keyfile_path = geth_keyfile(datadir, config["address"])
+                eth_create_account_file(keyfile_path, config["nodekey"])
+
     elif blockchain_type == "parity":
         genesis_path = os.path.join(base_datadir, "chainspec.json")
         parity_generate_chain_spec(
             genesis_path=genesis_path,
             genesis_description=genesis_description,
             seal_account=seal_account,
         )
-        parity_create_account(nodes_configuration[0], base_datadir, genesis_path)
```

Comment:
This was a bug, now all nodes which are miners will have their keys created. A future improvement may be to check that only one of the nodes is a miner for parity's PoA